python/cudf/cudf/utils/gpu_utils.py

```diff
@@ -0,0 +1,96 @@
+def validate_setup():
+    from cudf._cuda.gpu import (
+        getDeviceCount,
+        driverGetVersion,
+        runtimeGetVersion,
+        getDeviceAttribute,
+        CudaDeviceAttr,
+    )
+    import warnings
+
+    gpus_count = getDeviceCount()
+
+    if gpus_count > 0:
+        # Cupy throws RunTimeException to get GPU count,
+        # hence obtaining GPU count by in-house cpp api above
+
+        # 75 - Indicates to get "cudaDevAttrComputeCapabilityMajor" attribute
+        # 0 - Get GPU 0
+        major_version = getDeviceAttribute(
+            CudaDeviceAttr.cudaDevAttrComputeCapabilityMajor, 0
+        )
+
+        if major_version >= 6:
+            # You have a GPU with NVIDIA Pascal™ architecture or better
+            # Hardware Generation	Compute Capability
+            #    Turing	                7.5
+            #    Volta	                7.x
+            #    Pascal	                6.x
+            #    Maxwell	            5.x
+            #    Kepler	                3.x
+            #    Fermi	                2.x
+            pass
+        else:
+
+            warnings.warn(
+                "You will need a GPU with NVIDIA Pascal™ architecture or \
+                    better"
+            )
+
+        cuda_runtime_version = runtimeGetVersion()
+
+        if cuda_runtime_version > 10000:
+            # CUDA Runtime Version Check: Runtime version is greater than 10000
+            pass
+        else:
+            from cudf.errors import UnSupportedCUDAError
+
+            raise UnSupportedCUDAError(
+                "Please update your CUDA Runtime to 10.0 or above"
```

Comment:
Warn about what CUDA version was detected as opposed to just saying to update to 10.0 or above.

Reply:
Do we want to also print `cuda_runtime_version` in the error message? Or parse `cuda_runtime_version` to print the cuda version in error message? 
Asking this because `cuda_runtime_version` needs int to str conversion like..
CUDA 10.0 - cuda_runtime_version: 10000
CUDA 10.10 - cuda_runtime_version: 10010
CUDA 10.20 - cuda_runtime_version: 10020

In the docs I couldn't find the minor version is always going to be a double-digit number, else it would be a straight forward string split.

Reply:
We should parse it from these numbers and present it cleanly.

Reply:
Done, now the detected CUDA version will be printed.